packages/api/src/graphql/query.private.ts

```diff
@@ -415,7 +415,7 @@ export const GraphQLQuery = new GraphQLObjectType<undefined, Context>({
         const joins: SubscriptionJoins = {
           joinMemberPlan: true,
           joinPaymentMethod: true,
-          joinUser: true
+          joinUser: false
```

Comment:
@michael-scheurer so you identified that just the join of User schema is the problem right?

Reply:
correct.